lib/ansible/modules/monitoring/grafana_plugin.py

```diff
@@ -16,11 +16,12 @@
 ---
 module: grafana_plugin
 author:
-  - Thierry Sallé (@tsalle)
+  - Thierry Sallé (@seuf)
 version_added: "2.5"
 short_description: Manage Grafana plugins via grafana-cli
 description:
   - Install and remove Grafana plugins.
+  - See L(https://grafana.com/docs/plugins/installation/) for upstream documentation.
```

Comment:
Use `U(https://link)` syntax for URL links like this. The `L(...)` syntax requires two elements, `L(link_text,https://link)`. See https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#linking-within-module-documentation for more details.